### DISCUSSION TITLE
Pycharm output only in cpu mode by default

### DISCUSSION THREAD

**Discussion by arnoldpredator** [plasma-umass/scalene#311] [Q&A] [answered]
Hello folks, 

I just installed the "scalene" package and could output some html files. The problema I see is that there is no information on memory usage. It looks like it is only printing like in cpu-only mode. But I did not activate that option, this is the code I use:

scalene scratch_74.py  --html --outfile prof80.html

So, What could I do to get a more detailed report that includes memory, copy volume and other statistics?

Thank you!!!!!!

![Sin título](https://user-images.githubusercontent.com/32865392/147874983-329d2ab1-18e2-44ed-b19e-c0dc659317d0.png)

**Comment by emeryberger** [comment, marked as answer]
I'm guessing you are running Windows. Scalene produces all the relevant columns automatically (by default) on Linux and Mac OS X, but memory/copy volume and GPU profiling are not yet supported on Windows. Here's the output on my Mac.

<img width="1071" alt="Screen Shot 2022-01-02 at 10 34 36 AM" src="https://user-images.githubusercontent.com/1612723/147880791-39e5ce92-becf-4c84-9f53-38353e398f54.png">

**Reply by arnoldpredator** [reply]
Thanks Emery. I will try a virtual Linux on Windows then. 

Regards!

**Reply by emeryberger** [reply]
Just in case you mean running it on VMware or similar, Scalene also works fine on WSL2 (Windows Subsystem for Linux).

**Reply by arnoldpredator** [reply]
Nice, yes I meant that. I am going to try it right now.

I think with this tool and the book from Micha Gorelick y Ian Ozsvald "High Performance Python: Practical Performant Programming for Humans" I can learn a lot. If you know any other source that explains profiling I will be more than happy to read about it. 

Cheers!

edit:

I am reading it is possible to use the very same Windows software (Visual Studio code according to this guy https://www.youtube.com/watch?v=GvHc8KvoVIA) that I was using but with Linux in the background. Sounds really good. Would be amazing to see a tutorial on this. Hopefully I will manage by myself, let's see.

Apparently this is the tutorial I needed, just found it:

https://www.jetbrains.com/help/pycharm/using-wsl-as-a-remote-interpreter.html#create-wsl-interpreter

I put it here so it may help others. I am going to try.

edit again:

I think it is only available for Pycharm pro... I am going to try with VIM.
